src/patternfly/components/ChipGroup/examples/ChipGroup.md

```diff
@@ -12,39 +12,53 @@ cssPrefix: pf-c-chip-group
     {{#> chip-group-list chip-group-list--attribute='aria-label="Chip group list"'}}
       {{#> chip-group-list-item}}
         {{#> chip}}
-          {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_one_select_collapsed"')}}
-            Chip one
-          {{/chip-text}}
-          {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_one_select_collapsed ' chip-group--id 'chip_one_select_collapsed" aria-label="Remove" id="' chip-group--id 'remove_chip_one_select_collapsed"')}}
-            <i class="fas fa-times" aria-hidden="true"></i>
-          {{/button}}
+          {{#> chip-content}}
+						{{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_one_select_collapsed"')}}
+              Chip one
+            {{/chip-text}}
+					{{/chip-content}}
+          {{#> chip-actions}}
+						{{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_one_select_collapsed ' chip-group--id 'chip_one_select_collapsed" aria-label="Remove" id="' chip-group--id 'remove_chip_one_select_collapsed"')}}
+              <i class="fas fa-times" aria-hidden="true"></i>
+          	{{/button}}
+					{{/chip-actions}}
```

Comment:
nit - looks like you've got some indentation stuff going on in a lot of blocks in this file.